translations/Turkish/C/Cage/I Never Knew You/I Never Knew You

```diff
@@ -0,0 +1,50 @@
+# Translated into Turkish by https://twitter.com/GuerrillaFam
+# https://twitter.com/GuerrillaFam tarafından Türkçe'ye çevrilmiştir
```

Comment:
Could you please remove these two lines? Having text only before the metadata section is a huge improvement over other lyrics websites, besides URLs may be considered as advertisement, hence put the fair use status of this project in question, making it more likely to cause legal issues.

I'm currently planning to make two files in this repo, that will get rendered on the website: one with list of artists who donated lyrics to the project, one with list of contributors. I think that's the best that can be done without potentially getting into legal trouble and risking the project shutdown.

Reply:
I thought it's ok since I referenced [this file](https://github.com/Lyrics/lyrics/blob/master/translations/Esperanto/I/Isaac%20Watts/Joy%20to%20the%20world!/Joy%20to%20the%20World!), removed those lines 👍

Reply:
Ah, thanks for letting me know, will get rid of that part now, too. Will move his name into Credits.md to preserve authorship. Perhaps later we could add `Translated by` and `Transcribed by` metadata keys and let people leave their names within translations and transcribed texts, since it's sometimes important to know authorship from the text file itself, but I really worry about the impact it may have on the fair use status of the project, hence need to talk to a lawyer first.